### ISSUE TITLE
Support for proto3 `optional`

### ISSUE THREAD

**Issue by sradevski** [tokopedia/gripmock#72]
Hey there, great job on making this, it is much needed.

I tried using the package, but it seems `optional` is not supported. When running the package, I get `This file contains proto3 optional fields, but --experimental_allow_proto3_optional was not set.`, so maybe it's as simple as passing the experimental flag to `proto-gen`?

Cheers

**Comment by malins** [comment]
Just patch generator.go to also support optional keyword in proto files.

```Diff
diff --git a/protoc-gen-gripmock/generator.go b/protoc-gen-gripmock/generator.go
index cdd2258..5cb63b6 100644
--- a/protoc-gen-gripmock/generator.go
+++ b/protoc-gen-gripmock/generator.go
@@ -34,6 +34,7 @@ func main() {
        // Initialise our plugin with default options
        opts := protogen.Options{}
        plugin, err := opts.New(&request)
+       plugin.SupportedFeatures = uint64(pluginpb.CodeGeneratorResponse_FEATURE_PROTO3_OPTIONAL)
        if err != nil {
                log.Fatalf("error initializing plugin: %v", err)
        }
```

**Comment by mgerasimchuk** [comment]
Also faced:(

**Comment by sejin-P** [comment]
It's because of protoc in gripmock image is 3.12.2 which cannot support optional field. I tried to fix it, but all of my effort was unnecessary.

**Comment by ringerc** [comment]
Looks like PR https://github.com/tokopedia/gripmock/pull/119 is open for this, but unmerged/unreviewed with no comments.

**Comment by houcros** [comment]
I see that #119 was merged and (as far as I can tell) released in version 1.13. Does that mean that this issue is resolved and should be closed?

**Comment by jekiapp** [comment]
sorry, forgot to close. Released in 1.13
